src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.spec.ts

```diff
@@ -223,7 +226,8 @@ describe('LocalGameWrapperComponent', () => {
             const errorMessage: string = 'AI chose illegal move';
             const errorData: JSONValue = { name: 'P4', move: 'P4Move(0)' };
             expect(ErrorLoggerService.logError).toHaveBeenCalledWith('LocalGameWrapper', errorMessage, errorData);
-            tick(1000);
+            expect(messageDisplayer.criticalMessage).toHaveBeenCalledWith('The AI chose an illegal move! This is an unexpected situation that we logged, we will try to solve this as soon as possible.');
```

Comment:
"Meanwhile, consider that you won !" :p ? Humble & Friendly :D 

Sinon, le petit calcul du nombre de mot pour déterminer la durée de vie du popup va être utile pour celui là haha :D (vu que je crois qu'obliger l'user à cliquer pour libérer son écran est pas toujours user friendly)

Reply:
done & oui certainement !